fitbenchmarking/fitbenchmarking
# 742 flag minimizers not respecting bounds

#### Description of Work

Fixes #742 

#### Testing Instructions

1. Run fitbenchmarking and check that error flag appears as expected if parameter bounds are not respected (until pull request 771 is merged, using bumps minimizers are a good way to test this)

Function: Does the change do what it's supposed to?

Tests: Does it pass? Is there adequate coverage for new code?

Style: Is the coding style consistent? Is anything overly confusing?

Documentation: Is there a suitable change to documentation for this change?
